service/qos/service_level_controller.hh

```diff
@@ -259,13 +259,19 @@ public:
     /**
      * Gets the service level data by name.
      * @param service_level_name - the name of the requested service level
+     * @param or_default - in case the service level doesn't exist, if true
+     * returns the default service level, otherwise throws.
      * @return the service level data if it exists (in the local controller) or
      * get_service_level("default") otherwise.
      */
-    service_level& get_service_level(sstring service_level_name) {
+    service_level& get_service_level(sstring service_level_name, bool or_default) {
```

Comment:
Nit: Personally, I think it would be a bit more clear to have two separate functions rather than having one function controller by a boolean parameter. So: `get_service_level` and `get_service_level_or_default`, for example.

Reply:
done